packages/web/src/js/FileUploader.ts

```diff
@@ -0,0 +1,277 @@
+import { EventHandler, SelectorEngine } from './dom';
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+
+const NAME = 'fileUploader';
+const IS_DRAGGABLE_CLASS_NAME = 'has-drag-and-drop';
+const IS_DRAGGING_CLASS_NAME = 'is-dragging';
+const WRAPPER_ELEMENT_SELECTOR = '[data-element="wrapper"]';
```

Comment:
Can we use a `data-spirit` prefix please? E.g. there will be a `data-spirit-element` and `data-spirit-snippet`, etc.

But not `data-spirit-toggle` or `data-spirit-target`.

In the near future, we want to use a `spirit` prefix to avoid conflict in naming. Especially with the Bootstrap 4. So I think that we can use it right now with new components/selectors. It is nice to have but it will make it easier while we will undergoing the refactor.